lib/client/profile.go

```diff
@@ -34,19 +34,22 @@ const CurrentProfileSymlink = "profile"
 // type fewer CLI args.
 //
 type ClientProfile struct {
-	//
-	// proxy configuration
-	//
-	ProxyHost    string `yaml:"proxy_host,omitempty"`
-	ProxySSHPort int    `yaml:"proxy_port,omitempty"`
-	ProxyWebPort int    `yaml:"proxy_web_port,omitempty"`
+	// ProxyWebHost is the address of the web proxy can be accessed at.
+	ProxyWebHost string `yaml:"proxy_host,omitempty"`
+
+	// ProxyWebPort is port the web proxy can be accessed at.
+	ProxyWebPort int `yaml:"proxy_port,omitempty"`
+
+	// ProxySSHHost is the host the SSH proxy can be accessed at.
+	ProxySSHHost string `yaml:"proxy_ssh_host,omitempty"`
+
+	// ProxySSHPort is the port the SSH proxy can be accessed at.
+	ProxySSHPort int `yaml:"proxy_ssh_port,omitempty"`
```

Comment:
I would like this to be consistent with `kube_proxy_addr`, so `proxy_ssh_addr` and `proxy_addr` respectively.